### ISSUE TITLE
Heater runtime

### ISSUE THREAD

**Issue by Alek2ander** [tgstation/-tg-station#5607]
```
runtime error: Cannot modify null.update.
proc name: process (/obj/machinery/atmospherics/unary/heat_reservoir/process)
  source file: heat_source.dm,42
  usr: null
  src: the heater (/obj/machinery/atmospherics/unary/heat_reservoir/heater)
  call stack:
the heater (/obj/machinery/atmospherics/unary/heat_reservoir/heater): process()
the heater (/obj/machinery/atmospherics/unary/heat_reservoir/heater): process()
/datum/controller/game_control... (/datum/controller/game_controller): process machines()
/datum/controller/game_control... (/datum/controller/game_controller): process()
```

**Comment by duncathan** [comment]
I think this was fixed by #5850.

**Comment by phil235** [comment]
Yep
